src/main/scala/com/gu/mobile/content/notifications/lib/ContentAlertPayloadBuilder.scala

```diff
@@ -28,7 +28,9 @@ trait ContentAlertPayloadBuilder extends Logging {
 
   def buildPayLoad(content: Content): ContentAlertPayload = {
     val tagTypeSeries: Option[Tag] = content.tags.findOne(_.`type` == TagType.Series)
-    val tagTypeBlog: Option[Tag] = content.tags.findOne(_.`type` == TagType.Blog)
+    val tagTypeBlog: Option[Tag] = content.tags
+      .filterNot(tag => tag.sectionId.contains("commentisfree"))
```

Comment:
I think rather than `tag.sectionId` we may want to filter on the actual ID `tag.id` of that specific tag: `commentisfree/commentisfree`

Filtering on the sectionId means we may remove a legitimate blog that happen to have the section "commentisfree"

Reply:
Have amended to have `tag.id` instead